doc/tutorial-basic.md

```diff
@@ -3,9 +3,11 @@
 This tutorial should run without modification on Linux or Mac OS X,
 under [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
 
-You'll need about 5 GB of free disk space,
-and about 5 GB of RAM to search GenBank.  The tutorial should take about
-20 minutes total to run.
+You'll need about 5 GB of free disk space, and about 5 GB of RAM to
+search GenBank.  The tutorial should take about 20 minutes total to
+run. In fact, we have successfully tested it on
+[binder.pangeo.io](https://binder.pangeo.io/v2/gh/binder-examples/r-conda/master?urlpath=urlpath%3Drstudio)
```

Comment:
curious choice, why rstudio? =]

Reply:
oh, I've just been teaching with it and like the shell prompt there.